src/program/2023.ts

```diff
@@ -0,0 +1,182 @@
+      from: "2023-11-02T18:25:00",
+      to: "2023-11-02T19:15:00",
+      type: "info",
+    },
+    {
+      title: "Kom igang med åpen kildekode",
+      title_en: "Getting started with open source",
+      description:
+        "Hans Kristian Flaatten er plattform utvikler hos NAV, har brukt åpen kildekode hele sitt liv og vært en del av Node.js stiftelsen. I dette foredraget vil Hans Kristian snakke om sine erfaringer fra store og små åpen kildekode prosjekter om hvordan man kan komme i gang med bidrag eller til og med lage sine egne prosjekter.",
+      description_en:
+        "Hans Kristian Flaatten is a platform developer at NAV, has used open source all his life, and been part of the Node.js Foundation. In this talk, Hans Kristian will talk about his experiences from large and small open source projects on how to get started with contributions or even create your own projects.",
+      speaker: {
+        name: "Hans Kristian Flaatten",
+        description: "",
+      },
+      image: "TODO: path to some image in static dir",
+      from: "2023-11-02T19:20:00",
+      to: "2023-11-02T19:40:00",
+      type: "talk",
+      attachments: [
+        {
+          name: "Presentasjon",
+          name_en: "Presentation",
+          url: "/arkiv/2023/kom-i-gang.pdf",
+        },
+      ],
+      videoUrl:
+        "https://files.fribyte.no/boskonf/2023/Hans_Kristian_Flaatten_boskonf_2023.mp4",
+    },
+    {
+      title: "Eksperimentering for Grønnere Kode",
+      title_en: "Experimenting for Greener Code",
+      description:
+        "Kent Inge F. Simonsen er konsulent i Tietoevry og en av friBytes guruer. Kent Inge vil se på hvordan enkle eksperimenter kan hjelpe oss å bedre valg gitt målet om grønnere og mer effektiv kode.",
+      description_en:
+        "Kent Inge F. Simonsen is a consultant at Tietoevry and one of friBytes' gurus. Kent Inge will look at how simple experiments can help us make better choices given the goal of greener and more efficient code.",
+      speaker: {
+        name: "Kent Inge F. Simonsen",
+        description: "",
+      },
+      image: "TODO: path to some image in static dir",
+      from: "2023-11-02T19:45:00",
+      to: "2023-11-02T20:05:00",
+      type: "talk",
+      videoUrl:
+        "https://files.fribyte.no/boskonf/2023/Kent_Inge_F_Simonsen_boskonf_2023.mp4",
+    },
+    {
+      title: "Fest!🥳",
+      title_en: "Party!🥳",
+      description:
+        "Takk og farvel! For de som vil være med utover kvelden blir vi igjen på Kvarteret!",
+      description_en:
+        "Thank you and goodbye! For those who want to stay later in the evening, we'll be at Kvarteret!",
+      image: "TODO: path to some image in static dir",
+      from: "2023-11-02T20:05:00",
+      to: "2023-11-02T23:30:00",
+      type: "info",
+    },
+  ],
+  sponsors: [
+    {
+      name: "friByte",
+      logo: "/sponsors/friByte.svg",
+      url: "https://fribyte.no/",
+    },
+  ],
+};
```

Comment:
```suggestion
} as const;
```

This will make the type even easier to parse 🤓

Reply:
Hmm, what difference does this type casting make? I have `const program2023: Program = {` on top which defines the json object to be of type Program

Reply:
Probably won't make any difference 😎